docs/layouts/index.html

```diff
@@ -140,38 +140,13 @@
-                      <div class="item"> <blockquote><p>Finally, a publishing platform that's a joy to use. #NoMoreBarriers</p>&mdash; The Lone Cuber (@TheLoneCuber) <a href="https://twitter.com/TheLoneCuber/status/495731334711488512">August 2, 2014</a></blockquote> </div>
-                      <div class="item"> <blockquote><p>Hugo: Makes the Web Fun Again</p>&mdash; mercime (@mercime_one) <a href="https://twitter.com/mercime_one/status/500547145087205377">August 16, 2014</a></blockquote> </div>
-                      <div class="item"> <blockquote><p>Hugo is fast, dead simple to setup and well documented</p>&mdash; Markus Eliasson (@markuseliasson) <a href="https://twitter.com/markuseliasson/status/501594865877008384">August 19, 2014</a></blockquote> </div>
-                      <div class="item"> <blockquote><p>Finally the answer to the question my parents have been asking: What does Hugo do?</p>&mdash; Hugo Roy (@hugoroyd) <a href="https://twitter.com/hugoroyd/status/501704796727173120">August 19, 2014</a></blockquote> </div>
-                      <div class="item"> <blockquote><p>Also, I re-launched my blog (it looks the same as before) using Hugo, a *fast* static engine. Very happy with it.  <a href="http://gohugo.io/">gohugo.io</a></p>&mdash; Ryan Martinsen (@popthestack) <a href="https://twitter.com/popthestack/status/549972754125307904">December 30, 2014</a></blockquote> </div>
-                      <div class="item"> <blockquote><p>I migrated the <a href="https://twitter.com/BlackOpsTesting">@BlackOpsTesting</a>.com website from docpad to Hugo last weekend. http://gohugo.io/  Super Fast HTML Generation <a href="https://twitter.com/spf13">@spf13</a></p>&mdash; Alan Richardson (@eviltester) <a href="https://twitter.com/eviltester/status/553520335115808768">January 9, 2015</a></blockquote> </div>
-                      <div class="item"> <blockquote><p>Hugo is impressively capable. It's a static site generator by <a href="https://twitter.com/spf13">@spf13</a> written in <a href="https://twitter.com/hashtag/golang?src=hash">#golang</a>. Just upgraded to latest release; very powerful.</p>&mdash; Baron Schwartz (@xaprb) <a href="https://twitter.com/xaprb/status/556894866488455169">January 18, 2015</a></blockquote> </div>
-                      <div class="item"> <blockquote><p><a href="https://twitter.com/hashtag/Hugo?src=hash">#Hugo</a> A very good alternative for <a href="https://twitter.com/hashtag/wordpress?src=hash">#wordpress</a>!!! A fast and modern static website engine <a href="http://gohugo.io/">gohugo.io</a></p>&mdash; WorkHTML (@workhtml) <a href="https://twitter.com/workhtml/status/563064361301053440">February 4, 2015</a></blockquote> </div>
+                      {{ range $.Site.Data.references }}
```

Comment:
Oh, and the partial should be named quotes.html I guess, to be pedantic. As it expects a list of quotes.